advisor/src/main/kotlin/Advisor.kt

```diff
@@ -38,6 +38,7 @@ import org.ossreviewtoolkit.model.config.AdvisorConfiguration
 import org.ossreviewtoolkit.model.vulnerabilities.Vulnerability
 import org.ossreviewtoolkit.model.vulnerabilities.VulnerabilityReference
 import org.ossreviewtoolkit.plugins.api.PluginConfig
+import org.ossreviewtoolkit.plugins.api.orEmpty
```

Comment:
Maybe add https://github.com/oss-review-toolkit/ort/issues/9403#issuecomment-2582992199 as a reference to the commit message.